internal/field.go

```diff
@@ -104,7 +104,8 @@ func toCamelInitCase(name string, initUpper bool) string {
 			continue
```

Comment:
Did you try running the golang testsuite locally? Curious if there were any issues before we merge this.

Reply:
No issues, although the coverage seems fairly low.